libs/optimizer/src/lib/scoringRules.ts

```diff
@@ -0,0 +1,111 @@
+import { scoringRules as xcScoreRules} from "igc-xc-score";
+
+export enum ScoringRules {
+  CzechLocal,
+  CzechEuropean,
+  CzechOutsideEurope,
+  FederationFrancaiseVolLibre,
+  Leonardo,
+  Norway,
+  UnitedKingdomClub,
+  UnitedKingdomInternational,
+  UnitedKingdomNational,
+  XContest,
+  XContestPPG,
+  WorldXC,
+}
+
+const scoringBaseModel = xcScoreRules['XContest'];
```

Comment:
Let's discuss with Momtchill on how we can change this.
The indexes (below) might changed if the XContest rules get updated